units/src/amount/signed.rs

```diff
@@ -78,33 +123,33 @@ impl SignedAmount {
     ///
     /// # Errors
     ///
-    /// The function errors if the argument multiplied by the number of sats
-    /// per bitcoin overflows an `i64` type.
+    /// If `whole_bitcoin` is greater than `21_000_000`.
```

Comment:
I think `whole_bitcoin` is a terrible variable name.  I think `bitcoin_count` is more straightforward.

Reply:
Reviewing peoples PRs is kinda fun cause you can trash talk their code in the open xD.  I am trying to do it in a productive way though and hopefully you don't take offense.